op-node/Dockerfile

```diff
@@ -22,7 +23,7 @@ ARG TARGETOS TARGETARCH
 
 RUN make op-node VERSION="$VERSION" GOOS=$TARGETOS GOARCH=$TARGETARCH
 
-FROM alpine:3.16
+FROM alpine:latest
```

Comment:
I suggest specifying a version, using the latest one can easily lead to unpredictable issues, such as new versions having incompatible changes, which will silently be included in our release at some point.

Reply:
Yes. Update to alpine:3.16 like before. Please review again.